doc/sphinx/addendum/type-classes.rst

```diff
@@ -521,6 +559,20 @@ Settings
    the dependent ones previously (Coq 8.5 and below). This can result in
    quite different performance behaviors of proof search.
 
+   .. _TypeclassesDefaultMode:
+
+.. opt:: Typeclasses Default Mode {? ( {| "+" | "-" | "!" } ) }.
```

Comment:
Indeed, this doesn't make any sense. Please change it to:
```suggestion
.. opt:: Typeclasses Default Mode {| "+" | "-" | "!" }.
```

Reply:
Alright, I get it now.